pages/worker.i18n.json

```
@@ -76,18 +76,18 @@
     "テレワーク・時差出勤などスムーズビズの取組は、新型コロナウイルス感染症の対策としても効果的です。感染症対策として、東京2020大会時の交通混雑緩和に向けた取組の前倒しをお願いします。": "텔레워크·시차 출퇴근 등의 스무스 비즈 (Smooth Biz.) 의 노력은 신종 코로나 바이러스 감염 방지에도 효과적입니다. 감염 대책을 위해, 도쿄 2020 올림픽 때의 교통 혼잡 완화를 위한 조치를 앞당겨 시행하도록 부탁 드립니다."
   },
   "ja-basic": {
-    "企業の皆様・はたらく皆様へ": "はたらいている ひと へ",
-    "新型コロナウイルス感染症対応緊急融資": "コロナで つらい かいしゃや はたるくひと のために おかね を かします",
-    "新型コロナウイルスによる経営課題に関する専門家派遣": "コロナで かいしゃが つづけられるか こまっている ときに、かいしゃ の おかね や かいしゃで はたらく ひと の ことを いっしょに かんがえてくれる くわしい ひと が あなたの かいしゃに はなしを ききに いきます",
-    "事業継続緊急対策（テレワーク）助成金": "はたらく ひとが かいしゃではなく いえで はたらく ことに おかね を すこし だします",
-    "中小企業者等特別相談窓口": "とくに ちいさな かいしゃ の ひとが はなしたいとき の まどぐち",
-    "緊急労働相談ダイヤル": "かいしゃ や はたらく ひとが こまっているときの でんわ",
-    "新しいワークスタイルや企業活動の東京モデル「スムーズビズ」": "とうきょうと が おうえんしている あたらしい はたらきかた 「スムーズビズ」",
-    "新型コロナウイルス感染症により事業活動に影響を受けている中小企業等を対象とした緊急融資制度です。融資限度額は2億8千万円で、信用保証料を都が全額補助します。": "コロナ で かいしゃを つづけられるか こまっている とくに ちいさなかいしゃ のために おかねを かす しくみ です。２おく ８せんまんえん が おかね を かす いちばん おおきな きんがく です。しんようほしょう りょう という おかね は、すべて とうきょうと が ほしょう します。",
-    "新型コロナウイルス感染症により経営面の影響を受けている中小企業を対象に、中小企業診断士等の専門家を無料で派遣し、経営改善等に向けたアドバイスを実施します（1社あたり4回まで・無料）。": "コロナで かいしゃを つづけられるか こまっている とくに ちいさな かいしゃ のために、 かいしゃのおかねや かいしゃで はたるくひとの ことを いっしょに かんがえてくれる くわしい ひと が あなたの かいしゃに むかいます。（ひとつの かいしゃ に 4かい まで きてもらえます。おかねは かかりません。）",
-    "都内中堅・中小企業に対し、テレワークの導入に必要な機器やソフトウェア等の経費を助成します（限度額250万円、助成率10/10）。ぜひご活用ください。": "かいしゃに いかない で いえ で しごとを するために ひつような パソコン や パソコン ソフト を いれるのに かかる おかねを とうきょうと が すこし だします（250まん えん まで なら ぜんぶ の おかね をだします）",
-    "資金繰りに関する相談、経営に関する相談（東京都産業労働局 報道発表）": "かいしゃ を つづける ために ひつよう な まいにち の おかね で とても こまったときに はなすところ（とうきょうと さんぎょう ろうどうきょく が つたえて います）",
-    "新型コロナウイルスに関する休暇や休業の取り扱い、職場のハラスメントなどについての相談（東京都産業労働局 報道発表）": "コロナ の せいで（コロナ に かかっていなくても）すこし やすむ とき、しばらく やすむ とき、はたらいている ところで いやな こと を された と かんじた とき に はなす ところ（とうきょうと さんぎょう ろうどうきょく が つたえて います）",
-    "テレワーク・時差出勤などスムーズビズの取組は、新型コロナウイルス感染症の対策としても効果的です。感染症対策として、東京2020大会時の交通混雑緩和に向けた取組の前倒しをお願いします。": "いえではたらくこと じかんを ずらして かいしゃ に いくことは コロナに かかりづらく するために とても よいことです。2020とうきょう オリンピック パラリンピック のときに でんしゃや ばバス が こむ ように しない ためにも、 このような はたらきかた 「スムーズビズ」を はやめに やってみて ください"
+    "企業の皆様・はたらく皆様へ": "働（はたら）いている みなさん へ",
+    "新型コロナウイルス感染症対応緊急融資": "新（あたら）しい コロナウイルス感染症対応緊急融資（コロナウイルスかんせんしょうたいおう きんきゅうゆうし）：急（いそ）ぎ で お金（かね）を 貸（か）します",
+    "新型コロナウイルスによる経営課題に関する専門家派遣": "新（あたら）しい コロナウイルスによる 経営課題（けいえいかだい）に 関（かん）する 専門家（せんもんか）派遣（はけん）：会社（かいしゃ）を続けるのが 難しくなってきた ときに、会社（かいしゃ）の ことに 詳（くわ）しい 人（ひと） が 向（む）かいます",
+    "事業継続緊急対策（テレワーク）助成金": "事業継続緊急対策（じぎょうけいぞく きんきゅうたいさく）（テレワーク）助成金（じょせいきん）：会社（かいしゃ）を 続（つづ）けるために 家（いえ）で 働（はたら）く こと を 東京都（とうきょうと）が お金（かね）を 出（だ）して 応援（おうえん）します",
+    "中小企業者等特別相談窓口": "中小企業者等特別相談窓口（ちゅうしょうきぎょうしゃ とう とくべつそうだんまどぐち）：小（ちい）さな 会社（かいしゃ）の 相談（そうだん）に 特別（とくべつ）に 対応（たいおう）する ところ",
+    "緊急労働相談ダイヤル": "緊急労働相談ダイヤル（きんきゅう ろうどう そうだんダイヤル）：働く（ひと）の ための 相談（そうだん）電話（でんわ）",
```

```suggestion
    "緊急労働相談ダイヤル": "緊急労働相談ダイヤル（きんきゅう ろうどう そうだんダイヤル）：働（はたら）く人（ひと）の ための 相談（そうだん）電話（でんわ）",
```
ルビと親文字が対応していないようにみえますが、この修正で合ってるでしょうか？

---

あ、ほんとですね･･･。スプレッドシートが現在編集できないのですがこちらで先に修正して大丈夫ならこちらでお願いします。